src/css/styles.css

```
@@ -0,0 +1,74 @@
+/* Reset some basic elements (optional) */
```

Automated Code Review Suggestion:

```suggestion
.sidebar {
  width: 280px;
  background-color: #1c1c1c;
  min-height: 100vh; /* Fill vertical space */
  padding: 20px;
  border-right: 1px solid #333;
+ overflow-y: auto; /* Add vertical scrollbar if content overflows */
}
```
Added `overflow-y: auto;` to the `.sidebar` CSS class. This ensures that if the content within the sidebar exceeds the viewport height, a vertical scrollbar will appear, preventing content from overflowing and becoming inaccessible.  This improves the user experience by keeping all sidebar content visible.